comps/embeddings/multimodal_embeddings/multimodal_langchain/local_mm_embedding.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+from langsmith import traceable  # type: ignore
```

Comment:
please remove langsmith traceable

Reply:
Thank for your feedback. We have removed langsmith here.